common/scripts/lib/_parseArgs.sh

```diff
@@ -312,17 +312,14 @@ __set_host_services() {
   echo "HOST_SERVICES=true" >> "$ADMIRAL_ENV"
   source "$ADMIRAL_ENV"
 
-  ## add ssh key to host's authorized keys
-  local authorized_keys="/root/.ssh/authorized_keys"
-  local tmp_authorized_keys="/root/.ssh/tmp"
+  # create .ssh dir, if not present
+  local root_ssh_dir="/root/.ssh"
+  mkdir -p "$root_ssh_dir"
+
+  ## add public ssh key to host's authorized keys
+  local authorized_keys="$root_ssh_dir/authorized_keys"
   local ssh_public_key=$(cat "$SSH_PUBLIC_KEY")
-  # remove public key from authorized_keys, if present
```

Comment:
this is not required because adding same ssh key multiple times doesn't lead to any error. In centos, since the public ssh key is the only content of the `authorized_keys` file, inverse grep match (grep -v) returns error because grep doesn't find any other content to return. So, it exits with exit code 1. So instead of removing public key from authorized_keys, if present, we can just let public key to get added to authorized keys multiple times as it doesn't cause any error.